### PR TITLE
Preset refactor

### PR DESCRIPTION
**Pull request by ApocalyVec**

### CODE REVIEW

File: rena/ui/StreamOptionsWindow.py
Comment (HaowenWeiJohn) [inline rena/ui/StreamOptionsWindow.py:1]: get_nomimal_sampling_rate -> get_nominal_sampling_rate

Comment (HaowenWeiJohn) [inline rena/ui/StreamOptionsWindow.py:1]: The switching between other plotting profile  and Image will accumulate the Invalid Image Format Print  statement exponentially.
image

Comment (HaowenWeiJohn) [reply rena/ui/StreamOptionsWindow.py:1]: The self.dataDisplayDurationLineEdit has no effect to the Spectrogram plot
The spectrogram should automatically resize to the proper size based on the value in dataDisplayDurationLineEdit after the user switch to it.

**Review by HaowenWeiJohn — CHANGES_REQUESTED** [review]

File: rena/ui/StreamOptionsWindow.py
Comment (ApocalyVec) [reply rena/ui/StreamOptionsWindow.py:1]: Both issue fixed per [commit]https://github.com/ApocalyVec/RenaLabApp/pull/70/commits/6687b465a01f40e9221a2bb045a42383e2522370)

Comment (ApocalyVec) [reply rena/ui/StreamOptionsWindow.py:1]: typo fixed per 652831e4a4ce9832e2de4d24929c452f25f0aaa8

**Review by HaowenWeiJohn — APPROVED** [review]
Presets Refactor Reviewed

**Pull request merged**